### PR TITLE
Email spelling UPDATED

### PR DESCRIPTION
**Pull request by clementallen**
Following discussion I've re-named "email" to "Email" and "email Address / eMail Address" to "Email Address" across the files where necessary.

I've also changed the French translations to "e-mail address" as per <a href="http://www.collinsdictionary.com/dictionary/english-french/email-address?showCookiePolicy=true">Collins dictionary</a>

Also a slight grammar change to INSTALL.md

Hope I haven't missed any! :)

### CODE REVIEW

**Comment by clementallen** [comment]
Okay, only one test failed.  I'm on it.

**Comment by Turbo87** [comment]
hmm, you might have misunderstood me, sorry. I meant only changing it to capitalized "Email Address" when used in titles, otherwise "email address" is absolutely fine. I'll give some inline examples...

File: skylines/frontend/forms/pilot.py [hunk 1]
```diff
@@ -118,14 +118,14 @@ class LiveTrackingSettingsForm(Form):
 
 
 class RecoverStep1Form(Form):
-    email_address = EmailField(l_('eMail Address'), validators=[
-        InputRequired(message=l_('Please enter your email address.')),
+    email_address = EmailField(l_('Email Address'), validators=[
```
Comment (Turbo87) [inline skylines/frontend/forms/pilot.py]: this one is :+1:

File: skylines/frontend/forms/pilot.py [hunk 2]
```diff
@@ -118,14 +118,14 @@ class LiveTrackingSettingsForm(Form):
...
+        InputRequired(message=l_('Please enter your Email Address.')),
```
Comment (Turbo87) [inline skylines/frontend/forms/pilot.py]: this however is not a title and more like a sentence so I would just use "email address" in this case

**Pull request closed without merging**